src/lib/Components/Browse.php

```diff
@@ -30,7 +30,7 @@ public function renderToString()
         $selected = $this->getLocationId();
         // should be rendered with a twig template
         return '<ez-browse
-            class="ez-button"
+            class="ez-button  ez-button-action"
```

Comment:
It fixes the bug there :) I could put it in a dedicated commit, but since I am doing CSS on the buttons it's not a big deal. I would have done it if it hard to be backported.